benchmark/dns/lookup.js

```diff
@@ -5,7 +5,7 @@ const lookup = require('dns').lookup;
 
 const bench = common.createBenchmark(main, {
   name: ['', '127.0.0.1', '::1'],
-  all: [true, false],
+  all: ['true', 'false'],
```

Comment:
This actually breaks the intention of the Boolean. The value will be coerced to a Boolean again in line 15 and `!!'false' === true`. So you might just change this to `[1, 0]` instead.